test-packages/test-services/openapi/test-service/open-api.json

```diff
@@ -168,6 +167,120 @@
           "default"
         ]
       }
+    },
+    "/test-cases/parameters/required-parameters/{requiredPathItemPathParam}": {
```

Comment:
nice idea to seperate the test cases via pathes

Reply:
Yeah, it's still quite complex...